packages/commonwealth/test/unit/server_controllers/server_groups_controller.spec.ts

```diff
@@ -3,6 +3,28 @@ import { ServerGroupsController } from 'server/controllers/server_groups_control
 import { AddressInstance } from 'server/models/address';
 import { ChainInstance } from 'server/models/chain';
 import { UserInstance } from 'server/models/user';
+import { Requirement } from 'server/util/requirementsModule/requirementsTypes';
+
+const VALID_REQUIREMENTS: Requirement[] = [
+  {
+    rule: 'threshold',
+    data: {
+      threshold: '1000',
+      source: {
+        source_type: 'erc20',
+        chain_id: 'ethereum',
+        contract_address: '0x000',
+      },
+    },
+  },
+  {
+    rule: 'allow',
+    data: {
+      allow: ['0x1', '0x2'],
+    },
+  },
+];
+const INVALID_REQUIREMENTS = 'no an array' as unknown as [];
```

Comment:
Would like a little more thorough testing here, specifically for invalid requirements -- let's try to break it.